pkg/pushToNamespace/pushToServiceNamespaces.go

```diff
@@ -0,0 +1,45 @@
+package pushToNamespace
```

Comment:
We should change this package name to be compliant with [Go's guidelines for package names](https://go.dev/blog/package-names) and add the license header. Also it seems to me this package should go under `pkg/primaza/controlplane/` folder.

I'd propose to move this function directly in a new file in the package `pkg/primaza/controplane`, like `pkg/primaza/controlplane/servicenamespace.go`. What do you think?

Reply:
Yeah makes sense, anyways we were planning to move this to control plane latter, so shall do it now itself.